sipa/model/user.py

```diff
@@ -27,7 +27,7 @@ class AuthenticatedUserMixin:
     is_anonymous = False
 
 
-Row = namedtuple('Row', ['description', 'property'])
+Row = namedtuple('Row', [ 'property','description', 'subtext'])
```

Comment:
You can use a [`typing.NamedTuple`](https://docs.python.org/3/library/typing.html#typing.NamedTuple) with the advantage that you can specify a default Value of `None` for the `subtext`, then you don't have to add that to the dictionary every time in the parsing method below

Reply:
thanks thats great